generators/server/templates/src/main/resources/config/application.yml.ejs

```diff
@@ -368,7 +368,7 @@ jhipster:
     <%_ } _%>
 <%_ if (messageBroker === 'kafka') { _%>
 kafka:
-  bootstrap-servers: localhost:9092
+  bootstrap-servers: ${KAFKA_BOOTSTRAP_SERVERS:localhost:9092}
```

Comment:
This is not necessary, since the `KAFKA_BOOTSTRAP_SERVERS` environment variable should already have precedence over this entry config.

Reply:
Hmm are you sure of this when it's defined as an env var for the app service?